meta-xt-domx/recipes-extended/xen/xen-source.inc

```diff
@@ -1,6 +1,8 @@
 LIC_FILES_CHKSUM = "file://COPYING;md5=419739e325a50f3d7b4501338e44a4e5"
 
-SRC_URI = "git://github.com/xen-troops/xen.git;protocol=https;branch=xen-4.16rc-migration"
+XEN_URL ?= "git://github.com/xen-troops/xen.git;protocol=https;branch=xen-4.16rc-migration"
```

Comment:
make a sense

Reply:
I think ??= does not help here, because:
any "=" or "?=" assignment will override the value set with "??="

So, if variable has been defined during the parsing or our statements is not the latest one - '??=' does not help here.